mitiq/qse/qse.py

```diff
@@ -138,9 +141,9 @@ def qse_decorator(
         code_hamiltonian: Hamiltonian of the code space.
```

Comment:
Sorry can you explain why? It's a description of a variable `code_hamiltonian` which is indented by one level from the `Args` list. It looks like it renders correctly on the API-doc for me.

Reply:
It’s `check_operators` continuation that needs to be indented. GitHub wouldn’t let me comment on it directly because that line wasn’t changed in this PR.